src/nidigital/examples/nidigital_burst_with_start_trigger.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+
+import argparse
+import nidigital
+import os
+import sys
+
+
+def example(resource_name, options, source, rising_edge):
+
+    with nidigital.Session(resource_name=resource_name, options=options) as session:
+
+        dir = os.path.dirname(__file__)
+
+        # Configure the session by loading pin map (.pinmap) created in the Digital Pattern Editor on the instrument
+        pin_map_filename = os.path.join(dir, 'burst_with_start_trigger_files', 'PinMap.pinmap')
+        session.load_pin_map(pin_map_filename)
+
+        # Configure the session by loading the specifications (.sepcs), levels (.digilevels), and timing (.digitiming) files created in the Digital Pattern Editor on the instrument
+        spec_filename = os.path.join(dir, 'burst_with_start_trigger_files', 'Specifications.specs')
+        levels_filename = os.path.join(dir, 'burst_with_start_trigger_files', 'PinLevels.digilevels')
+        timing_filename = os.path.join(dir, 'burst_with_start_trigger_files', 'Timing.digitiming')
+        session.load_specifications_levels_and_timing(spec_filename, levels_filename, timing_filename)
+
+        # Apply the settings from the levels and timing files we just loaded on the instrument.
+        session.apply_levels_and_timing(levels_filename, timing_filename)
+
+        # Configure the session by loading the pattern file (.digipat) created in the Digital Pattern Editor on the instrument.
+        pattern_filename = os.path.join(dir, 'burst_with_start_trigger_files', 'Pattern.digipat')
+        session.load_pattern(pattern_filename)
+
+        # Specify a source and edge for the external start trigger
+        session.digital_edge_start_trigger_source = source
+        session.digital_edge_start_trigger_edge = nidigital.DigitalEdge.RISING if rising_edge else nidigital.DigitalEdge.FALLING
+
+        # Waiting for the trigger to start bursting and then blocks until the pattern is done bursting
+        session.burst_pattern('new_pattern')
+
+        # Disconnect all channels using programmable onboard switching
+        session.selected_function = nidigital.SelectedFunction.DISCONNECT
+
```

Comment:
When the example is run there is no indication of success/fail. Consider logging a couple of lines to the console.